app/jobs/channels/approve_channel_transfer_job.rb

```diff
@@ -11,7 +11,7 @@ def perform(channel_id:)
         ## There is a helper function in channel.rb that can help minimize this code
         ## Need to generate new bF deposit ID on channel transfer:
         channel.deposit_id = nil
-        if contested_by.publisher.selected_wallet_provider_type == BITFLYER_CONNECTION
+        if contested_by.publisher.selected_wallet_provider_type == Channel::BITFLYER_CONNECTION
```

Comment:
quick q, does Channel::BITFLYER_CONNECTION return 'BitflyerConnection'

Reply:
Yeah it does. 

```
  BITFLYER_CONNECTION = "BitflyerConnection".freeze
```